fhevm/tee_arthmetic_gas.go

```diff
@@ -39,3 +39,41 @@ func teeMulRequiredGas(environment EVMEnvironment, input []byte) uint64 {
 
 	return environment.FhevmParams().GasCosts.TeeMul[lhs.fheUintType()]
 }
+
+func teeDivRequiredGas(environment EVMEnvironment, input []byte) uint64 {
+	input = input[:minInt(65, len(input))]
+
+	logger := environment.GetLogger()
+
+	var lhs, rhs *verifiedCiphertext
+	lhs, rhs, err := get2VerifiedOperands(environment, input)
+	if err != nil {
+		logger.Error("teeDiv RequiredGas() ciphertext inputs not verified", "err", err, "input", hex.EncodeToString(input))
+		return 0
+	}
+	if lhs.fheUintType() != rhs.fheUintType() {
+		logger.Error("teeDiv RequiredGas() operand type mismatch", "lhs", lhs.fheUintType(), "rhs", rhs.fheUintType())
+		return 0
+	}
+
+	return environment.FhevmParams().GasCosts.TeeDiv[lhs.fheUintType()]
```

Comment:
It seems like these codes are also repeated. Any ways to refactor them?

Reply:
Okay. I'll try.